lib/http-bridge/test/integration/Main.hs

```diff
@@ -96,24 +98,35 @@ main = do
         let stateDir = "./test/data/cardano-node-simple"
         let networkDir = "/tmp/cardano-http-bridge/networks"
         let bridgePort = 8080
+        let nodeApiAddress = "127.0.0.1:3101"
         removePathForcibly (networkDir <> "/local")
         handle <-
             openFile "/tmp/cardano-wallet-launcher" WriteMode
-        systemStart <-
-            formatTime defaultTimeLocale "%s" . addUTCTime 5 <$> getCurrentTime
-        cluster <- async $ void $ launch
-            [ cardanoNodeSimple stateDir systemStart ("core0", "127.0.0.1:3000")
-            , cardanoNodeSimple stateDir systemStart ("core1", "127.0.0.1:3001")
-            , cardanoNodeSimple stateDir systemStart ("core2", "127.0.0.1:3002")
-            , cardanoNodeSimple stateDir systemStart ("relay", "127.0.0.1:3100")
-            , cardanoHttpBridge bridgePort"local" networkDir handle
+        start <-
+            formatTime defaultTimeLocale "%s" . addUTCTime 2 <$> getCurrentTime
+        [h0, h1, h2, h3] <- forM ["core0", "core1", "core2", "relay"] $ \x -> do
+            openFile ("/tmp/cardano-node-simple/" <> x) WriteMode
+        cluster <- async $ throwIO =<< launch
+            [ cardanoNodeSimple stateDir start ("core0", "127.0.0.1:3000") h0 []
+            , cardanoNodeSimple stateDir start ("core1", "127.0.0.1:3001") h1 []
+            , cardanoNodeSimple stateDir start ("core2", "127.0.0.1:3002") h2 []
+            , cardanoNodeSimple stateDir start ("relay", "127.0.0.1:3100") h3
+                [ "--node-api-address", nodeApiAddress
+                , "--node-doc-address", "127.0.0.1:3102"
+                , "--tlscert", "/dev/null"
+                , "--tlskey", "/dev/null"
+                , "--tlsca", "/dev/null"
+                , "--no-tls"
```

Comment:
Apparently, even if we provide `--no-tls`, we still have to give all the `--tls***` options to the node... undoubtedly nice CLI we have there.